### PR TITLE
Changed register button from teal to orange

### PR DESCRIPTION
**Pull request by jkga**
Makes the "register" button more prominent.

### CODE REVIEW

**Comment by tjmonsi** [comment]
Cannot merge this. This is not part of the design guidelines. Please connect with @juvarabrera first and get approval from him on this (show screen shot and create an issue)

Will close until gets approved

**Comment by juvarabrera** [comment]
Approved pull request.

File: src/pages/page-home/index.js [hunk 1]
```diff
@@ -47,6 +48,7 @@ class Page extends PageMixin(ElementLiteLit(HTMLElement, style.toString())) {
   }
 
   get about () {
+    updateState('currentProgress', 'loaded');
```
Comment (tjmonsi) [inline src/pages/page-home/index.js]: what is this for?

Can you show pictures (i'm just lazy running it right now)

Comment (tjmonsi) [reply src/pages/page-home/index.js]: @jkga

**Comment by tjmonsi** [comment]
@juvarabrera ok copy will wait for @jkga to show some pictures to me so that I know before even merging.

File: src/pages/page-home/index.js (hunk 1 above)
Comment (jkga) [reply src/pages/page-home/index.js]: That is for hiding the progress bar when page is loaded. It is inspired by [gdgph's](https://www.gdgph.org/#/) progress bar .You can [view it here](https://searca-my.sharepoint.com/:i:/g/personal/jkga_searca_org/EURMSvPPncVAlEUYu4tUDUMBW2Pwcz3IPyqvMvlt3rRX2g?e=cE8HO2)

**Comment by tjmonsi** [comment]
@jkga next time please use "spaces = 2" on indentation. Thanks

**Comment by jkga** [comment]
@tjmonsi noted. Thanks!

**Pull request merged**